genTriposGAFFandGAFF2.py

```diff
@@ -1,20 +1,19 @@
 #!/usr/bin/env python
 
-import os
+# Purpose: Generate Tripos/GAFF/GAFF2 mol2 files for purposes of
+#  comparing minimizations with different force fields. This script
+#  generates input files for all the molecules listed in the single input file.
+# Usage: python file.py -i /path/to/input.sdf -l /the/output/dir
```

Comment:
@dkyu I would usually put a section like this in the triple quotes (""") you and @vtlim  should also list yourselves as authors somewhere.